servatrice/src/servatrice.cpp

```diff
@@ -543,27 +549,51 @@ void Servatrice::statusUpdate()
     servatriceDatabaseInterface->execSqlQuery(query);
 
     // send activation emails
```

Comment:
I waffle with this, I can see your point but the code itself lies inside a function already (the statusUpdate()) .  Unless the recommended practices have changed (as they do over times and I am old), I was taught to not call functions from other functions as jumping through functions to finish tasks is troublesome to troubleshoot *unless* the function is used else were in the code (in this case it is not).